frontend/device/android/device.lua

```diff
@@ -375,7 +377,22 @@ function Device:canExecuteScript(file)
     end
 end
 
+function Device:lightDialog()
+    local usleep = require("ffi/util").usleep
+    android.settings.dialog(_("Frontlight settings"), _("Intensity"), _("Warmth"), _("Ok"))
+    repeat
+        usleep(75000) -- sleep 75ms before next check if dialog was quit
+    until (not android.isFrontlightDialogRunning())
+    logger.info("new brightness: " .. self.powerd.fl_intensity)
+    self.powerd:setIntensityHW(self.powerd:frontlightIntensityHW())
+    if android.isWarmthDevice() then
+        self.powerd:setWarmth(self.powerd:getWarmth())
+        logger.info("new warmth: " .. self.powerd.fl_warmth)
+    end
+end
+
 android.LOGI(string.format("Android %s - %s (API %d) - flavor: %s",
     android.prop.version, getCodename(), Device.firmware_rev, android.prop.flavor))
 
 return Device
+
```

Comment:
Intended?

Reply:
poire-z asked me in https://github.com/koreader/koreader/pull/6385 to add a `\n` at end of file.

Reply:
That's technically two of 'em ;).

(i.e. we definitely want a LF after `Device`, but then you have another one on L398).

Reply:
OK I changed that, it a peculiarity of my editor?